types/api.go

```diff
@@ -28,11 +28,11 @@ type MetaRequest struct {
 	Content      []byte     `json:"content,omitempty"`
 	Digested     bool       `json:"digested,omitempty"`
 	EntityId     HexBytes   `json:"entityId,omitempty"`
-	From         int64      `json:"from,omitempty"`
-	FromID       HexBytes   `json:"fromId,omitempty"`
-	ListSize     int64      `json:"listSize,omitempty"`
+	From         int        `json:"from,omitempty"`
+	ListSize     int        `json:"listSize,omitempty"`
```

Comment:
removing FromID is a breaking change, right?

Also, I think using `int` instead of `int64` for the public API type is a bad idea. the API will have different integer precision depending on the machine that's running the server. you want it to be fixed, so int64, int32, uint64, etc.

Reply:
I used "int" because that's the same type used by badgerhold for the Skip() statement. I think that is the most correct approach since if the machine have int=64b then both badgerhold and the API will use the same type.